python/desc/imsim/cosmic_rays.py

```diff
@@ -88,12 +89,12 @@ def paint_cr(self, image_array, index=None, pixel=None):
         numpy.array: The input image array with the CR added.
         """
         if index is None:
-            cr = random.choice(self)
+            cr = self.rng.choice(self)
```

Comment:
I'm sure this a python feature I don't appreciate, but is there a reason this isn't

self.rng.choice(self.image_array)
 
and then self.image_array[index]

?

Reply:
OK Jim explained to me that CosmicRay inherits from list so self is also a list.

Also in my comment above image_array wouldn't be the correct choice since that is where the CRs are going.